### PR TITLE
UI improvments and general panel fixes

### PR DESCRIPTION
**Pull request by thewander02**
user profile, networking, backups, Schedules

### CODE REVIEW

File: resources/scripts/components/dashboard/AccountOverviewContainer.tsx [hunk 1]
```diff
@@ -1,17 +1,25 @@
...
+import { Tabs, TabsContent, TabsList, TabsTrigger } from '@/components/elements/Tabs';
 
 import Code from '../elements/Code';
 
 export default () => {
     const { state } = useLocation();
 
+    const user = useStoreState((state: State<ApplicationStore>) => state.user.data!);
+    const emailHash = sha256(user.email).toString();
+    const avatar = `https://www.gravatar.com/avatar/${emailHash}?s=128&d=identicon`;
```
Comment (itsvic-dev) [inline resources/scripts/components/dashboard/AccountOverviewContainer.tsx:21]: no

Comment (denisapain) [reply resources/scripts/components/dashboard/AccountOverviewContainer.tsx:21]: how to turn off gravatar on my instance of pyro

Comment (prplwtf) [reply resources/scripts/components/dashboard/AccountOverviewContainer.tsx:21]: gravatar is for when you don't want to deal with uploadable profile pictures
i do not want to deal with uploadable profile pictures

Comment (denisapain) [reply resources/scripts/components/dashboard/AccountOverviewContainer.tsx:21]: i do want to deal with uploadable pictures so how turn it offs

Comment (denisapain) [reply resources/scripts/components/dashboard/AccountOverviewContainer.tsx:21]: > > gravatar is for when you don't want to deal with uploadable profile pictures
> > i do not want to deal with uploadable profile pictures
> 
> Real

REMUV GAYVATAR RIGHT NOW OR ELSE I NUKE CHINA

**Comment by ferothefox** [comment]
Thank you for your contribution! We'll merge the changes to the settings page in a future push, but we won't merge the changes that include Gravatar as it doesn't align with our internal plans for the account system.

**Comment by thewander02** [comment]
Do you want me to make a pull with out gravatar and just a placeholder div so you can just merge? I have it ready on my fork if so

**Comment by prplwtf** [comment]
_getting contributions on random organizations: part one_

**Pull request closed without merging**